tests/files/enki/empty_item_circulation.xml

```diff
@@ -0,0 +1 @@
+<ArrayOfItemCirculation xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" />
```

Comment:
This is a Bibliotheca response, which makes me think this possibility isn't tested.